iota/transaction/creation.py

```diff
@@ -302,10 +399,17 @@ def add_inputs(self, inputs):
     def send_unspent_inputs_to(self, address):
         # type: (Address) -> None
         """
-        Adds a transaction to send "change" (unspent inputs) to the
-        specified address.
+        Specifies the address that will receive unspent IOTAs.
```

Comment:
```suggestion
        Specifies the address that will receive unspent iotas.
```

(I'll just keep suggesting `iotas` for now 😅)